pyproject.toml

```diff
@@ -27,6 +27,7 @@ matplotlib = "^3.7.0"
 pyserde = "^0.9.8"
 polars = "^0.19.15"
 scikit-learn = "^1.4.0"
+pheval-template = "^0.1.1"
```

Comment:
does this need to be a dependency? Or could it be pip installed into the environment?

Reply:
No, it doesn't. I will install it with pip in QC workflow.

Reply:
Enough as "dev" dependency?